src/Writer/XLSX/Options.php

```diff
@@ -63,4 +71,65 @@ public function getMergeCells(): array
     {
         return $this->MERGE_CELLS;
     }
+
+    /**
+     * set Worksheets page margins.
+     *
+     * @param float $top    inches
+     * @param float $right  inches
+     * @param float $bottom inches
+     * @param float $left   inches
+     * @param float $header inches
+     * @param float $footer inches
+     */
+    public function setPageMargin(
+        float $top = 0.75,
+        float $right = 0.7,
+        float $bottom = 0.75,
+        float $left = 0.7,
+        float $header = 0.3,
+        float $footer = 0.3
+    ): void {
+        $this->pageMargin = new PageMargin(
+            $top,
+            $right,
+            $bottom,
+            $left,
+            $header,
+            $footer,
+        );
+    }
+
+    public function getPageMargin(): PageMargin
```

Comment:
Returning `?PageMargin` seems enough for me to the rid of `hasPageMargin` method

Reply:
yup, i agree with that, but the phpstan will throw an error.
```
Only booleans are allowed in an if condition, OpenSpout\Writer\XLSX\Options\PageMargin given
```

Reply:
```php
$pageMargin = $options->getPageMargin();
if (null !== $pageMargin) {
    // ...
}
```

Reply:
i think i'll just use isset()